### ISSUE TITLE
No Mouse Input on Nightly Builds in Fedora 40

### ISSUE THREAD

**Issue by JustPlainGarak** [LizardByte/Sunshine#2971]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

Mouse input from clients (I have tested multiple client devices, all on the latest stable Moonlight AppImage build) does not move the mouse on the host PC running Fedora 40. This includes both movement and clicks.

The issue is present both in the automated builds available from github as well as a source built RPM from the latest master branch.

This issue is not present in v0.23.1 built from the master_legacy branch on my machine.

### Expected Behavior

When I interact with the mouse on the client, those actions should occur on the host.

### Additional Context

I enabled debug logging to see if I could find any errors, and as far as I can tell, there are none. I've put some of the log into the field below but will attach the full log file as well in the event that my cherry picking doesn't reveal something that the full log file does.
[sunshine-no-mouse-f40-v2024.805.184417-debug.log](https://github.com/user-attachments/files/16513225/sunshine-no-mouse-f40-v2024.805.184417-debug.log)

### Host Operating System

Linux

### Operating System Version

Fedora Linux 40 (KDE Plasma)

### Architecture

64 bit

### Sunshine commit or version

https://github.com/LizardByte/Sunshine/commit/4bd521bb4360d8ba9b1af8d2d704e8f79c3e9a38

### Package

Linux - rpm

### GPU Type

AMD

### GPU Model

AMD Radeon RX 7900 XTX

### GPU Driver/Mesa Version

Mesa 24.1.5

### Capture Method

KMX (Linux)

### Config

```Shell
min_threads = 4
qp = 25
fec_percentage = 40
min_fps_factor = 10
audio_sink = alsa_output.usb-M-Audio_AIR_192_4-00.analog-stereo
fps = [30,60,90,120,144,240]
nv_preset = default
amd_quality = default
amd_rc = auto
sw_preset = veryfast
sunshine_name = EmpokNor
channels = 2
nv_rc = auto
key_rightalt_to_key_win = enabled
origin_pin_allowed = lan
resolutions = [
    1280x720,
    1920x1080,
    2560x1440
]
output_name = 2
```

### Relevant log output

```shell
[2024:08:06:08:46:49]: Debug: ---Begin Response---
RTSP/1.0 200 OK
CSeq: 7



---End Response---
[2024:08:06:08:46:49]: Debug: Initialized new control stream session by connect data match [v2]
[2024:08:06:08:46:49]: Debug: Control local address [10.4.20.69]
[2024:08:06:08:46:49]: Debug: Control peer address [10.4.20.111:44750]
[2024:08:06:08:46:49]: Info: CLIENT CONNECTED
[2024:08:06:08:46:49]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2024:08:06:08:46:49]: Debug: type [IDX_START_B]
[2024:08:06:08:46:49]: Debug: RAISE: 10.4.20.111:46689 :: VIDEO
[2024:08:06:08:46:49]: Debug: Received ping [v2] from 10.4.20.111:46689 [0100000038304534434233354238463143304638]
[2024:08:06:08:46:49]: Debug: Start capturing Video
[2024:08:06:08:46:49]: Info: /dev/dri/card1 -> amdgpu
[2024:08:06:08:46:49]: Debug: Opening render node: /dev/dri/renderD128
[2024:08:06:08:46:49]: Debug: crtc(0, 0)
[2024:08:06:08:46:49]: Debug: crtc(1920, 1080)
[2024:08:06:08:46:49]: Debug: plane->possible_crtcs == 4
[2024:08:06:08:46:49]: Debug: x(0) y(0) crtc_x(0) crtc_y(0) crtc_id(100)
[2024:08:06:08:46:49]: Debug: Resolution: 1920x1080: Pitch: 8192: Offset: 0
[2024:08:06:08:46:49]: Debug: Format [XR24, AR24, RA24, XR30, XB30, AR30, AB30, XR48, XB48, AR48, AB48, XB24, AB24, RG16, NV12, P010, XR4H, AR4H, XB4H, AB4H]
[2024:08:06:08:46:49]: Debug: crtc(0, 0)
[2024:08:06:08:46:49]: Debug: crtc(2560, 1440)
[2024:08:06:08:46:49]: Debug: plane->possible_crtcs == 2
[2024:08:06:08:46:49]: Debug: x(0) y(0) crtc_x(0) crtc_y(0) crtc_id(96)
[2024:08:06:08:46:49]: Debug: Resolution: 2560x1440: Pitch: 10240: Offset: 0
[2024:08:06:08:46:49]: Debug: Format [XR24, AR24, RA24, XR30, XB30, AR30, AB30, XR48, XB48, AR48, AB48, XB24, AB24, RG16, NV12, P010, XR4H, AR4H, XB4H, AB4H]
[2024:08:06:08:46:49]: Debug: crtc(0, 0)
[2024:08:06:08:46:49]: Debug: crtc(2560, 1440)
[2024:08:06:08:46:49]: Debug: plane->possible_crtcs == 1
[2024:08:06:08:46:49]: Debug: x(0) y(0) crtc_x(0) crtc_y(0) crtc_id(92)
[2024:08:06:08:46:49]: Debug: Resolution: 2560x1440: Pitch: 10240: Offset: 0
[2024:08:06:08:46:49]: Debug: Format [XR24, AR24, RA24, XR30, XB30, AR30, AB30, XR48, XB48, AR48, AB48, XB24, AB24, RG16, NV12, P010, XR4H, AR4H, XB4H, AB4H]
[2024:08:06:08:46:49]: Info: Found display [wayland-0]
[2024:08:06:08:46:49]: Debug: Available interface: wl_compositor(1) version 6
[2024:08:06:08:46:49]: Debug: Available interface: zwp_tablet_manager_v2(3) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_keyboard_shortcuts_inhibit_manager_v1(4) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zxdg_decoration_manager_v1(5) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_viewporter(6) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_security_context_manager_v1(7) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_fractional_scale_manager_v1(8) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wl_shm(9) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wl_seat(10) version 9
[2024:08:06:08:46:49]: Debug: Available interface: zwp_pointer_gestures_v1(11) version 3
[2024:08:06:08:46:49]: Debug: Available interface: zwp_pointer_constraints_v1(12) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_relative_pointer_manager_v1(13) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wl_data_device_manager(14) version 3
[2024:08:06:08:46:49]: Debug: Available interface: zwlr_data_control_manager_v1(15) version 2
[2024:08:06:08:46:49]: Debug: Available interface: wp_cursor_shape_manager_v1(16) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_primary_selection_device_manager_v1(17) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_idle(18) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_idle_inhibit_manager_v1(19) version 1
[2024:08:06:08:46:49]: Debug: Available interface: ext_idle_notifier_v1(20) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_plasma_shell(21) version 8
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_appmenu_manager(22) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_server_decoration_palette_manager(23) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_plasma_virtual_desktop_management(25) version 2
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_shadow_manager(27) version 2
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_dpms_manager(28) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_server_decoration_manager(29) version 1
[2024:08:06:08:46:49]: Debug: Available interface: kde_output_management_v2(30) version 9
[2024:08:06:08:46:49]: Debug: Available interface: zxdg_output_manager_v1(31) version 3
[2024:08:06:08:46:49]: Info: Found interface: zxdg_output_manager_v1(31) version 3
[2024:08:06:08:46:49]: Debug: Available interface: wl_subcompositor(32) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zxdg_exporter_v2(33) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zxdg_importer_v2(34) version 1
[2024:08:06:08:46:49]: Debug: Available interface: xdg_activation_v1(37) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_content_type_manager_v1(39) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_tearing_control_manager_v1(40) version 1
[2024:08:06:08:46:49]: Debug: Available interface: xdg_toplevel_drag_manager_v1(41) version 1
[2024:08:06:08:46:49]: Debug: Available interface: kde_screen_edge_manager_v1(42) version 1
[2024:08:06:08:46:49]: Debug: Available interface: frog_color_management_factory_v1(43) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wp_presentation(44) version 1
[2024:08:06:08:46:49]: Debug: Available interface: xdg_wm_dialog_v1(45) version 1
[2024:08:06:08:46:49]: Debug: Available interface: xdg_wm_base(48) version 6
[2024:08:06:08:46:49]: Debug: Available interface: zwlr_layer_shell_v1(49) version 5
[2024:08:06:08:46:49]: Debug: Available interface: wp_drm_lease_device_v1(51) version 1
[2024:08:06:08:46:49]: Debug: Available interface: kde_output_order_v1(52) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_text_input_manager_v1(53) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_text_input_manager_v2(54) version 1
[2024:08:06:08:46:49]: Debug: Available interface: zwp_text_input_manager_v3(55) version 1
[2024:08:06:08:46:49]: Debug: Available interface: wl_drm(57) version 2
[2024:08:06:08:46:49]: Debug: Available interface: zwp_linux_dmabuf_v1(58) version 4
[2024:08:06:08:46:49]: Debug: Available interface: wp_linux_drm_syncobj_manager_v1(59) version 1
[2024:08:06:08:46:49]: Debug: Available interface: kde_output_device_v2(60) version 8
[2024:08:06:08:46:49]: Debug: Available interface: kde_output_device_v2(61) version 8
[2024:08:06:08:46:49]: Debug: Available interface: kde_output_device_v2(62) version 8
[2024:08:06:08:46:49]: Debug: Available interface: wl_output(63) version 4
[2024:08:06:08:46:49]: Info: Found interface: wl_output(63) version 4
[2024:08:06:08:46:49]: Debug: Available interface: wl_output(64) version 4
[2024:08:06:08:46:49]: Info: Found interface: wl_output(64) version 4
[2024:08:06:08:46:49]: Debug: Available interface: wl_output(65) version 4
[2024:08:06:08:46:49]: Info: Found interface: wl_output(65) version 4
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_blur_manager(66) version 1
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_contrast_manager(67) version 2
[2024:08:06:08:46:49]: Debug: Available interface: org_kde_kwin_slide_manager(68) version 1
[2024:08:06:08:46:49]: Info: Resolution: 1920x1080
[2024:08:06:08:46:49]: Info: Resolution: 2560x1440
[2024:08:06:08:46:49]: Info: Resolution: 2560x1440
[2024:08:06:08:46:49]: Info: Offset: 640x0
[2024:08:06:08:46:49]: Info: Logical size: 1920x1080
[2024:08:06:08:46:49]: Info: Name: HDMI-A-1
[2024:08:06:08:46:49]: Info: Found monitor: Samsung Electric Company C27F591/HTHJ805097
[2024:08:06:08:46:49]: Info: Offset: 2560x1080
[2024:08:06:08:46:49]: Info: Logical size: 2560x1440
[2024:08:06:08:46:49]: Info: Name: DP-1
[2024:08:06:08:46:49]: Info: Found monitor: Samsung Electric Company LC32G7xT/H4ZN801004
[2024:08:06:08:46:49]: Info: Offset: 0x1080
[2024:08:06:08:46:49]: Info: Logical size: 2560x1440
[2024:08:06:08:46:49]: Info: Name: DP-2
[2024:08:06:08:46:49]: Info: Found monitor: Dell Inc. DELL U2711/D971T2B110PL
[2024:08:06:08:46:49]: Info: -------- Start of KMS monitor list --------
[2024:08:06:08:46:49]: Info: Monitor 0 is HDMI-A-1: Samsung Electric Company C27F591/HTHJ805097
[2024:08:06:08:46:49]: Info: Monitor 2 is DP-1: Samsung Electric Company LC32G7xT/H4ZN801004
[2024:08:06:08:46:49]: Info: Monitor 1 is DP-2: Dell Inc. DELL U2711/D971T2B110PL
[2024:08:06:08:46:49]: Info: --------- End of KMS monitor list ---------
[2024:08:06:08:46:49]: Debug: Monitor description
[2024:08:06:08:46:49]: Debug: Resolution: 0x0
[2024:08:06:08:46:49]: Debug: Offset: 0x0
[2024:08:06:08:46:49]: Debug: Monitor description
[2024:08:06:08:46:49]: Debug: Resolution: 2560x1440
[2024:08:06:08:46:49]: Debug: Offset: 2560x1080
[2024:08:06:08:46:49]: Debug: Monitor description
[2024:08:06:08:46:49]: Debug: Resolution: 2560x1440
[2024:08:06:08:46:49]: Debug: Offset: 0x1080
[2024:08:06:08:46:49]: Debug: Monitor description
[2024:08:06:08:46:49]: Debug: Resolution: 1920x1080
[2024:08:06:08:46:49]: Debug: Offset: 640x0
[2024:08:06:08:46:49]: Debug: Desktop resolution: 5120x2520
[2024:08:06:08:46:49]: Info: Screencasting with KMS
[2024:08:06:08:46:49]: Info: /dev/dri/card1 -> amdgpu
[2024:08:06:08:46:49]: Debug: Opening render node: /dev/dri/renderD128
[2024:08:06:08:46:49]: Info: Found monitor for DRM screencasting
[2024:08:06:08:46:49]: Debug: crtc(0, 0)
[2024:08:06:08:46:49]: Debug: crtc(2560, 1440)
[2024:08:06:08:46:49]: Debug: plane->possible_crtcs == 1
[2024:08:06:08:46:49]: Debug: x(0) y(0) crtc_x(0) crtc_y(0) crtc_id(92)
[2024:08:06:08:46:49]: Debug: Resolution: 2560x1440: Pitch: 10240: Offset: 0
[2024:08:06:08:46:49]: Debug: Format [XR24, AR24, RA24, XR30, XB30, AR30, AB30, XR48, XB48, AR48, AB48, XB24, AB24, RG16, NV12, P010, XR4H, AR4H, XB4H, AB4H]
[2024:08:06:08:46:49]: Info: Found connector ID [106]
[2024:08:06:08:46:49]: Info: Found cursor plane [89]
[2024:08:06:08:46:49]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:49]: Debug: EGL: [Mesa Project]: version [1.5]
[2024:08:06:08:46:49]: Debug: API's supported: [OpenGL OpenGL_ES ]
[2024:08:06:08:46:49]: Debug: GL: vendor: AMD
[2024:08:06:08:46:49]: Debug: GL: renderer: AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 18.1.6, DRM 3.57, 6.9.12-203.fsync.fc40.x86_64)
[2024:08:06:08:46:49]: Debug: GL: version: 4.6 (Compatibility Profile) Mesa 24.1.5
[2024:08:06:08:46:49]: Debug: GL: shader: 4.60
[2024:08:06:08:46:49]: Info: HDR color coding [Rec. 2020 + SMPTE 2084 PQ]
[2024:08:06:08:46:49]: Info: Color depth: 10-bit
[2024:08:06:08:46:49]: Info: Color range: [JPEG]
[2024:08:06:08:46:49]: Info: vaapi vendor: Mesa Gallium driver 24.1.5 for AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 18.1.6, DRM 3.57, 6.9.12-203.fsync.fc40.x86_64)
[2024:08:06:08:46:49]: Debug: [AVHWDeviceContext @ 0x7f07282b8080] VAAPI driver: Mesa Gallium driver 24.1.5 for AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 18.1.6, DRM 3.57, 6.9.12-203.fsync.fc40.x86_64).
[2024:08:06:08:46:49]: Debug: [AVHWDeviceContext @ 0x7f07282b8080] Driver not found in known nonstandard list, using standard behaviour.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f0728238300] Input surface format is p010le.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f0728238300] Using VAAPI profile VAProfileAV1Profile0 (32).
[2024:08:06:08:46:49]: Error: [av1_vaapi @ 0x7f0728238300] No usable encoding entrypoint found for profile VAProfileAV1Profile0 (32).
[2024:08:06:08:46:49]: Info: Retrying with fallback configuration options for [av1_vaapi] after error: Function not implemented
[2024:08:06:08:46:49]: Info: vaapi vendor: Mesa Gallium driver 24.1.5 for AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 18.1.6, DRM 3.57, 6.9.12-203.fsync.fc40.x86_64)
[2024:08:06:08:46:49]: Debug: [AVHWDeviceContext @ 0x7f0728245380] VAAPI driver: Mesa Gallium driver 24.1.5 for AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 18.1.6, DRM 3.57, 6.9.12-203.fsync.fc40.x86_64).
[2024:08:06:08:46:49]: Debug: [AVHWDeviceContext @ 0x7f0728245380] Driver not found in known nonstandard list, using standard behaviour.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Input surface format is p010le.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Using VAAPI profile VAProfileAV1Profile0 (32).
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Using VAAPI entrypoint VAEntrypointEncSlice (6).
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Using VAAPI render target format YUV420_10 (0x100).
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] RC mode: CBR.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] RC target: 100% of 6508000 bps over 11 ms.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] RC buffer: 72311 bits, initial fullness 54233 bits.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] RC framerate: 90/1 (90.00 fps).
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Driver does not report any additional prediction constraints.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Using intra and P-frames (supported references: 1 / 0).
[2024:08:06:08:46:49]: Warning: [av1_vaapi @ 0x7f07282b3d00] Multiple slices were requested but this codec does not support controlling slices.
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] All wanted packed headers available (wanted 0x3, found 0x3).
[2024:08:06:08:46:49]: Debug: [av1_vaapi @ 0x7f07282b3d00] Using level 4.1.
[2024:08:06:08:46:49]: Debug: Minimum frame time set to 100ms, based on min fps factor of 1.
[2024:08:06:08:46:49]: Debug: Frame 1: IDR Keyframe (AV_FRAME_FLAG_KEY)
[2024:08:06:08:46:49]: Debug: Sent HDR mode: true
[2024:08:06:08:46:49]: Debug: --begin relative mouse move packet--
deltaX [1]
deltaY [1]
--end relative mouse move packet--
[2024:08:06:08:46:49]: Debug: --begin relative mouse move packet--
deltaX [-1]
deltaY [-1]
--end relative mouse move packet--
[2024:08:06:08:46:49]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2024:08:06:08:46:49]: Debug: Frame 18: IDR Keyframe (AV_FRAME_FLAG_KEY)
[2024:08:06:08:46:49]: Debug: --begin controller arrival packet--
controllerNumber [0]
type [00]
capabilities [0043]
supportedButtonFlags [0000F7FF]
--end controller arrival packet--
[2024:08:06:08:46:49]: Info: Gamepad 0 will be Xbox One controller (default)
[2024:08:06:08:46:50]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2024:08:06:08:46:50]: Debug: Frame 35: IDR Keyframe (AV_FRAME_FLAG_KEY)
[2024:08:06:08:46:50]: Debug: RAISE: 10.4.20.111:43149 :: AUDIO
[2024:08:06:08:46:50]: Debug: Received ping [v2] from 10.4.20.111:43149 [0200000038304534434233354238463143304638]
[2024:08:06:08:46:50]: Debug: Start capturing Audio
[2024:08:06:08:46:50]: Debug: Connecting to pulseaudio
[2024:08:06:08:46:50]: Debug: null-sink args: rate=48000 sink_name=sink-sunshine-stereo format=float channels=2 channel_map=front-left,front-right sink_properties=device.description=sink-sunshine-stereo
[2024:08:06:08:46:50]: Debug: null-sink args: rate=48000 sink_name=sink-sunshine-surround51 format=float channels=6 channel_map=front-left,front-right,front-center,lfe,rear-left,rear-right sink_properties=device.description=sink-sunshine-surround51
[2024:08:06:08:46:50]: Debug: null-sink args: rate=48000 sink_name=sink-sunshine-surround71 format=float channels=8 channel_map=front-left,front-right,front-center,lfe,rear-left,rear-right,side-left,side-right sink_properties=device.description=sink-sunshine-surround71
[2024:08:06:08:46:50]: Info: Setting default sink to: [sink-sunshine-stereo]
[2024:08:06:08:46:50]: Info: Found default monitor by name: alsa_output.usb-M-Audio_AIR_192_4-00.analog-stereo.monitor
[2024:08:06:08:46:50]: Info: Opus initialized: 48 kHz, 2 channels, 96 kbps (total), LOWDELAY
[2024:08:06:08:46:50]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2024:08:06:08:46:50]: Debug: Frame 51: IDR Keyframe (AV_FRAME_FLAG_KEY)
[2024:08:06:08:46:50]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2024:08:06:08:46:50]: Debug: Frame 53: IDR Keyframe (AV_FRAME_FLAG_KEY)
[2024:08:06:08:46:50]: Debug: System tray loop
[2024:08:06:08:46:51]: Debug: METHOD :: GET
[2024:08:06:08:46:51]: Debug: DESTINATION :: /api/logs
[2024:08:06:08:46:51]: Debug: Priority -- u=4
[2024:08:06:08:46:51]: Debug: Host -- empoknor:47990
[2024:08:06:08:46:51]: Debug: Accept -- */*
[2024:08:06:08:46:51]: Debug: Sec-GPC -- 1
[2024:08:06:08:46:51]: Debug: User-Agent -- Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0
[2024:08:06:08:46:51]: Debug: Accept-Language -- en-US,en;q=0.5
[2024:08:06:08:46:51]: Debug: DNT -- 1
[2024:08:06:08:46:51]: Debug: Referer -- https://empoknor:47990/troubleshooting
[2024:08:06:08:46:51]: Debug: Accept-Encoding -- gzip, deflate, br, zstd
[2024:08:06:08:46:51]: Debug: Connection -- keep-alive
[2024:08:06:08:46:51]: Debug: Authorization -- CREDENTIALS REDACTED
[2024:08:06:08:46:51]: Debug: Sec-Fetch-Dest -- empty
[2024:08:06:08:46:51]: Debug: Sec-Fetch-Mode -- cors
[2024:08:06:08:46:51]: Debug: Sec-Fetch-Site -- same-origin
[2024:08:06:08:46:51]: Debug:  [--] 
[2024:08:06:08:46:51]: Debug:  [--] 
[2024:08:06:08:46:53]: Debug: --begin relative mouse move packet--
deltaX [-1]
deltaY [0]
--end relative mouse move packet--
[2024:08:06:08:46:53]: Debug: --begin relative mouse move packet--
deltaX [-1]
deltaY [1]
--end relative mouse move packet--
[2024:08:06:08:46:53]: Debug: --begin relative mouse move packet--
deltaX [-1]
deltaY [0]
--end relative mouse move packet--
...
--end relative mouse move packet--
[2024:08:06:08:46:57]: Debug: --begin relative mouse move packet--
deltaX [2]
deltaY [1]
--end relative mouse move packet--
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:57]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:58]: Debug: Refreshing cursor image after FB changed
[2024:08:06:08:46:59]: Debug: --begin relative mouse move packet--
deltaX [0]
deltaY [1]
--end relative mouse move packet--
```

**Comment by ReenigneArcher** [comment]
Is the user running sunshine part of the input group?

**Comment by JustPlainGarak** [comment]
> Is the user running sunshine part of the input group?

Yep!

`[JustPlainGarak@EmpokNor~]% cat /etc/group | grep input`
`input:x:104:JustPlainGarak`

**Comment by ReenigneArcher** [comment]
I assume it's related to this change: https://github.com/LizardByte/Sunshine/pull/2606

**Comment by JustPlainGarak** [comment]
> I assume it's related to this change: #2606

Oh, that could certainly be the case. I'll swap back to the pre-release build and see if the udev rule and uhid module load files get added with the installation. If they DON'T get added, I'll create them myself, reboot and see what the result is. Either way, I'll report the results of that process here once completed.

**Comment by JustPlainGarak** [comment]
Unfortunately, I am still seeing the same behavior. I updated to the Aug 5th build again, set the udev rule, set the module to load at boot and rebooted the computer.

While the uhid module loaded (confirmed with lsmod and /proc/modules), no mouse input registers on the host PC. Keyboard and controller input continue to work as they did previously.

**Comment by gschintgen** [comment]
You seem to be comfortable building Sunshine from source. You could try building it once after checking out commit 509576d61608925deb6f2360a5f5cba9521a71fb
(merge commit of the new input system) and once with the previous commit. That should be 5f1a17f77d4bc2f29d4d7b6bfc9f0e0b913fcddb.

For automatic loading of the uhid module see #2906

**Comment by JustPlainGarak** [comment]
> You seem to be comfortable building Sunshine from source. You could try building it once after checking out commit [509576d](https://github.com/LizardByte/Sunshine/commit/509576d61608925deb6f2360a5f5cba9521a71fb) (merge commit of the new input system) and once with the previous commit. That should be [5f1a17f](https://github.com/LizardByte/Sunshine/commit/5f1a17f77d4bc2f29d4d7b6bfc9f0e0b913fcddb).
> 
> For automatic loading of the uhid module see #2906

Here are my results from testing the two commits:

- commit **509576d**: mouse input does not work
- commit **5f1a17f**: mouse input works
 
I think that confirms the general suspicion that something in the new input system merged in with **509576d** is breaking virtual mouse input, at least on my system.

I'd already setup the module auto-load for uhid yesterday and confirmed the module is (and was during testing) loaded with lsmod and /proc/modules (at least I'm pretty sure that's what these outputs mean). Here's the output from them for reference.
```
sudo lsmod | grep uhid
uhid                   24576  1
sudo cat /proc/modules | grep uhid
uhid 24576 1 - Live 0x<what I assume is a memory address>
```

**Comment by ABeltramo** [comment]
`uhid` is just used for emulating the PS5 joypad with Gyro and acceleration,  everything else uses `/dev/uinput`.

A few questions:
 - Can you confirm if it's just the mouse or does the keyboard work? 
 - Can you run `udevadm monitor` before starting the stream and check which events (if any) gets triggered by starting the stream? 
 - Can you install `evemu-record` (it's under evdev-tools I believe on Debian, not sure under Fedora) and check if whilst running a Sunshine stream it lists additional virtual devices and it does receive any input?
Specifically, which udev rules have you installed?

**Comment by JustPlainGarak** [comment]
Alright, well...

I just spent the last 2 hours trying to reproduce the issue, gathered logs for the requests from @ABeltramo and all that jazz...but the issue is gone and I can't make it come back. I thought that maybe, somehow, installing evemu had fixed the issue, but removing the package had no effect: my mouse simply works now. This isn't only with the latest nightly, I also installed the August 5th nightly I was testing with before and all inputs, mouse included, work fine now.

I can only assume that there was some now resolved upstream issue in Fedora that silently fixed this issue for me as I have no other explanation. For the sake of completeness, I will provide/confirm the following:

My udev rules:
```
[JustPlainGarak@EmpokNor~]% cat /etc/udev/rules.d/60-sunshine.rules
KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess"
KERNEL=="uhid", TAG+="uaccess"
```
The issue was only with the mouse. The keyboard and any/all controllers I threw at it worked just fine.

Thanks to everyone for the time and effort looking into this issue with me. I hate that I don't have root cause for us, but hey...at least the problem is gone?
